cognite_toolkit/_cdf_tk/commands/featureflag.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+import tempfile
+from enum import Enum
+from functools import lru_cache
+from pathlib import Path
+from typing import Any, ClassVar
+
+import yaml
+from rich import print
+from rich.table import Table
+
+from cognite_toolkit._cdf_tk.commands._base import ToolkitCommand
+from cognite_toolkit._cdf_tk.exceptions import ToolkitRequiredValueError
+
+
+class Flags(Enum):
+    INTERACTIVE_INIT: ClassVar[dict[str, Any]] = {"visible": True, "description": "Enables interactive init"}
+    INTERNAL: ClassVar[dict[str, Any]] = {"visible": False, "description": "Does nothing"}
+
+
+class FeatureFlag:
+    @staticmethod
+    def _get_file() -> Path:
+        f = Path(tempfile.gettempdir()) / "tk-ff.bin"
+        if not f.exists():
+            f.write_text("{}")
+        return f
+
+    @staticmethod
+    def _load_user_settings() -> dict[str, bool]:
+        return yaml.safe_load(FeatureFlag._get_file().read_text())
+
+    @staticmethod
+    def _save_user_settings(flag: Flags, enabled: bool) -> None:
+        settings = FeatureFlag._load_user_settings()
+        settings.update({flag.name: enabled})
+        FeatureFlag._get_file().write_text(yaml.dump(settings))
+        FeatureFlag.is_enabled.cache_clear()
+
+    @staticmethod
+    def _reset_user_settings() -> None:
+        FeatureFlag._get_file().unlink()
+        FeatureFlag.is_enabled.cache_clear()
+
+    @staticmethod
+    @lru_cache(typed=True)
+    def is_enabled(flag: str | Flags) -> bool:
+        if isinstance(flag, str):
+            fflag = FeatureFlag.to_flag(flag)
+        else:
+            fflag = flag
+
+        if not fflag:
+            return False
+
+        user_settings = FeatureFlag._load_user_settings()
+        return user_settings.get(fflag.name, False)
+
+    @staticmethod
+    @lru_cache
+    def to_flag(flag: str) -> Flags | None:
+        flag = flag.upper()
+        for member in Flags:
+            if member.name == flag:
+                return member
+        return None
```

Comment:
Enums have a getitem implemented for this
```suggestion
        try:
            return Flags[flag.upper()]
        except KeyError:
            return None
```